### ISSUE TITLE
Documentation update: environment variable replacing

### ISSUE THREAD

**Issue by sebastianst** [tmux-python/tmuxp#719]
The documentation about [environment variable replacing](https://tmuxp.git-pull.com/examples.html#environment-variable-replacing) needs to be update to include more fields in which env vars are substituted. E.g. this also happens for variables defined under `environment:`.

**Comment by tony** [comment]
@sebastianst Agreed on this. I welcome a PR, if there's no pull request I will get around to a more comprehensive overview of variable replacement.
